orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/dcos/DcosContainerFinder.groovy

```diff
@@ -20,7 +20,10 @@ class DcosContainerFinder {
       if (!image) {
         throw new IllegalStateException("No image found in context for pattern $imageDescription.pattern.")
       } else {
-        imageDescription = [registry: image.registry, tag: image.tag, repository: image.repository, imageId: buildImageId(image.registry, image.repository, image.tag)]
```

Comment:
The imageId that was being used in clouddriver was the original imageId, it was acting like the assignment never happened.